src/collectd.conf.in

```diff
@@ -1215,6 +1216,12 @@
 #	</Process>
 #</Plugin>
 
+#<Plugin "procevent">
+#  BufferLength 10
+#  RegexProcess "/^ovs.*$/" 
```

Comment:
I think naming should be `ProcessRegex`, what do you think?

Reply:
Agreed.  The change would make for better naming.